app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -896,12 +903,14 @@ class PipelineSampleReport extends React.Component {
           aria-hidden="true"
         />
       );
-    if (reportDetails.assembled_taxids.indexOf(taxInfo.tax_id.toString()) >= 0)
-      assemblyDot = (
+    if (taxInfo.tax_id > 0 && taxInfo.tax_level == 1 && taxInfo.NT.r > 0)
+      // right now, tree pipeline is only implemented for tax_level 1, but may change in future
+      phyloTreeDot = (
         <i
           data-tax-id={taxInfo.tax_id}
-          onClick={this.downloadAssemblyLink}
-          className="fa fa-gg"
+          data-tax-level={taxInfo.tax_level}
+          onClick={this.gotoTreeLink}
```

Comment:
It is better to have explicit calls: `onClick={()=> this.goToTreeLink(taxInfo)}`

Reply:
ok